hack/header.txt

```diff
@@ -1,15 +1,15 @@
-/*
-Copyright YEAR (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-     http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+// Copyright (C) YEAR SAP SE or an SAP affiliate company. All rights reserved.
```

Comment:
Replace `YEAR` with `2019`

Reply:
Then we are good to go

Reply:
It's templating: `YEAR` gets automatically replaced with `2019` (current year)